packages/server/src/services/carpool.service.ts

```diff
@@ -43,6 +43,19 @@ export class CarpoolService {
         return carpool;
     }
 
+    /**
+     * Finds a carpool by its URL ID.
+     * @param urlId - URL ID of the carpool
+     */
+    public async findOneByUrlId(urlId: string): Promise<Carpool> {
```

Comment:
I know this is stupid, but ... why is this method name formatted as `findOneBy..` when everything else in the class is `carpool`-specific?

Reply:
Haha not stupid! We should stick to a consistent naming convention, so I'll update that to be `findCarpoolByUrlId`.